core/src/mindustry/game/Rules.java

```diff
@@ -189,6 +189,8 @@
     public float backgroundOffsetX = 0.1f, backgroundOffsetY = 0.1f;
     /** Parameters for planet rendered in the background. Cannot be changed once a map is loaded. */
     public @Nullable PlanetParams planetBackground;
+    /** Rules from this planet are applied. If it's {@code sun}, mixed tech is enabled. */
+    public Planet planet = Planets.serpulo;
```

Comment:
Extremely misleading, as `Vars.state.rules.planet` in the campaign will return  `null`.

Reply:
I've made this apply to campaign mode as well instead. 🤷‍♂️